src/ext/java/org/opentripplanner/ext/flex/FlexLegMapper.java

```diff
@@ -30,5 +30,6 @@ static public void fixFlexTripLeg(Leg leg, FlexTripEdge flexTripEdge) {
       leg.alightRule = GraphPathToItineraryMapper.getBoardAlightMessage(3);
 
       leg.bookingInfo = flexTripEdge.getFlexTrip().getBookingInfo(leg.from.stopIndex);
+      leg.generalizedCost = flexTripEdge.getTimeInSeconds();
```

Comment:
It was intentional, but is not needed :-)

Since Flex paths are mapped using `State`s, `generalizedCost` is already filled out with `state.getWeight()`.